Legacy/real_time/arduino_src/radio_buggy_mega/Makefile

```diff
@@ -467,51 +489,46 @@ extcoff: $(TARGET).elf
 
 
 # Link: create ELF output file from object files.
-.SECONDARY : $(TARGET).elf

```

Comment:
I'm not sure what this line does, I have been trying to compile with and without and haven't seen a difference